packs/trendmicro-cloudone-conformity/config.yml

```
@@ -28,8 +28,18 @@ authors:
 
 documentation:
   - name: GitHub repository
+    description: The GitHub repository for the integration
     url: https://github.com/newrelic-experimental/newrelic-connector-trendmicro-cloud-one-conformity#readme
   - name: New Relic blog post
+    description: Learn more about the integration on New Relic's blog post
     url: https://newrelic.com/blog/how-to-relic/trend-micro-aws
   - name: Trend Micro blog post
+    description: description: Learn more about the integration on Trend Micro's blog post
```

The extra `description:` here is invalidating the yaml when the CI checks attempt to read it. 
```suggestion
    description: Learn more about the integration on Trend Micro's blog post
```